.github/workflows/back-to-development.yaml

```diff
@@ -0,0 +1,30 @@
+name: Go back to Development
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+env:
+  APP_ID: 251311
+
+jobs:
+  post-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+      - name: Get GitHub App Token
+        uses: actions/create-github-app-token@v1
```

Comment:
This is needed because we want a CI check to check that this PR has changes in CHANGELOG.md. To trigger that check, this PR needs to be opened with an App token.